addons/point_of_sale/models/pos_session.py

```diff
@@ -1397,12 +1397,13 @@ def _amount_converter(self, amount, date, round):
         return self.currency_id._convert(amount, self.company_id.currency_id, self.company_id, date, round=round)
 
     def show_cash_register(self):
```

Comment:
This method is now unused. But since you properly adapted the returned value, perhaps you can restore the button you removed from [pos_session_view.xml](https://github.com/odoo/odoo/pull/99092/files#diff-97e241e39c63c662bf535d48566bc0c8f26724826b79faec4af6c9b67148cb9e)?

Reply:
The problem is we don't have this view (tree view of transactions) in the community version, it is now an enterprise only. If you think this button is necessary, I can add it in the enterprise, if you think it is not that important I can erase this method for good

Reply:
Would be nice to keep it in enterprise so that it will look like we didn't remove a feature. Would you agree @pimodoo? Or can we just let the button go?

Reply:
As it exists, it could be put in enterprise. By saying it only exists in enterprise, do you mean it is in the module account_accountant? because we don't depend on it. So if it means we should do create a module pos_account_accountant, we probably should not keep it.

Reply:
Yes it is in account_accountant